django/text_extractor/utils.py

```diff
@@ -133,10 +133,22 @@ def dist(p1, p2):
 
 
 def create_searchable_pdf(input_file, add_header):
+    # Reset the file pointer to the beginning
+    input_file.seek(0)
+    file_content = input_file.read()
+    print(f"Input file size: {len(file_content)} bytes")
+
     # Create a temporary file and write the contents of the uploaded file to it
+
+    # without celery:
```

Comment:
Keep these comments incase we want to test it without celery in future for debugging purposes